stored_procedures/create_run_record.sql

```diff
@@ -62,7 +59,7 @@ BEGIN
     now()
   );
 
-  SELECT LAST_INSERT_ID() INTO @automationSystemRunId;
+  SELECT LAST_INSERT_ID() INTO output_automation_system_run_id;
```

Comment:
Does it need an `@`/ what does `@` do?

Reply:
The examples I saw have the @ for internal (local) variables but not for the IN / OUT parameters. So think that's correct but not certain.